### PR TITLE
Add support for Ubuntu Bionic

### PR DESCRIPTION
**Pull request by Kami**
This pull request adds support for Ubuntu Bionic.

In addition to that, it includes some other changes / fixes:

* Invalid syntax in rules file
* Out of date Dockerfile.template

### CODE REVIEW

**Comment by Kami** [comment]
@armab We need to decide if we will continue to use Dockerfile.template and update.py stuff.

It seems like changes got added directly to Dockerfiles and not the template which got out of date. If we plan to continue to use it, we will need to hook it up to a CI or similar.

Same story for st2packagingbuild-dockerfiles repo.

File: debian/rules [hunk 1]
```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/make -f
-IS_SYSTEMD = $(shell command -v dh_systemd_enable > /dev/null 2>&1 && echo true)
+IS_SYSTEMD := $(shell command -v dh_systemd_enable > /dev/null 2>&1 && echo true)
 
 %:
-	dh $@ $(shell $IS_SYSTEMD && echo --with=systemd)
```
Comment (Kami) [inline debian/rules:5]: This syntax was wrong:

```bash
...
debian/rules clean
/bin/sh: 1: S_SYSTEMD: not found
dh clean 
   dh_auto_clean
	make -j1 clean
make[1]: Entering directory '/root/st2chatops'
rm -Rf node_modules/
mkdir -p node_modules/
make[1]: Leaving directory '/root/st2chatops'
   dh_clean
 debian/rules build
/bin/sh: 1: S_SYSTEMD: not found
dh build 
   dh_update_autotools_config
   dh_auto_configure
   dh_auto_build
	make -j1
...
```

**Comment by Kami** [comment]
@armab Any idea why those steps are failing - https://circleci.com/gh/StackStorm/st2chatops/618#build-parameters/containers/3?

screenshot from 2018-11-07 15-40-08

Afaik, I didn't touch any of that.

**Comment by Kami** [comment]
I believe Bionic build package steps fails because this change hasn't been merged yet (https://github.com/StackStorm/st2-packages/pull/592), but no idea why builds for other distros fail.

**Comment by Kami** [comment]
I believe Bionic build package step fails because this PR hasn't been merged yet (https://github.com/StackStorm/st2-packages/pull/592) but no idea why other jobs fail.

**Comment by Kami** [comment]
Please ignore my comment above about st2box related failures - @blag already fixed that in his PR (#117) and I cherry picked those changes here - 5762591.

**Comment by Kami** [comment]
Synced this branch up with latest master.

Using Node.js 10 also allowed me to get rid of the conditional from Bionic Node.js install since default Ubuntu repos ship with 8 and we use 10 so 10 will be used during install.

**Review by arm4b — APPROVED** [review]
:+1: 

Also looks really clean now, after merging that another PR with CI fixes.

**Pull request merged**